internal/images/fastjpeg/fastjpeg.go

```diff
@@ -213,9 +213,18 @@ func DecodeDownsample(r io.Reader, factor int) (image.Image, error) {
 	stderrW := new(bytes.Buffer)
 	cmd.Stderr = stderrW
 	if err := cmd.Run(); err != nil {
+		// if djpeg finished with warnings ( exit code 2 )
+		// the output is still usable most of the time
```

Comment:
shouldn't you at least look at the `w` `bytes.Buffer` to see if it was written to and is the expected size?